changelog/1.38.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "1.38.0"
+description: "Released on 01/04/2023"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.38.0.
+
+### Features ✨
+
+- Individual image tags can now be decommissioned. Existing workspaces using a
+  decommissioned tag will be migrated to use the default tag upon rebuild. Adds
+  new organization-level roles: Organization Super Manager, Organization Image
+  Manager, and Organization Importer. These roles allow finer-grained access
+  control around image and registry management. For more information, see
+  Organization Roles.
```

Comment:
In case we want to add the link:

```suggestion
  [Organization Roles](https://coder.com/docs/coder/latest/admin/access-control/organizations).
```